### ISSUE TITLE
GEP: Define Gateway API Interaction with Multicluster Services

### ISSUE THREAD

**Issue by robscott** [kubernetes-sigs/gateway-api#1748]
**What would you like to be added**:
We need to define via documentation how Gateway API interacts with Multicluster Services. This will likely be a docs-only change, but may also include a change to API Spec to note "ServiceImport" is supported for "extended" conformance.

**Why this is needed**:
There are already implementations of Multi-Cluster Gateway either released or in progress, it would be great to ensure that we're all providing a consistent experience.

**Comment by costinm** [comment]
I assume this can also cover GAMMA ( both as backend reference, but also attaching to ServiceImport in addition to Service as a frontend).

**Comment by robscott** [comment]
> I assume this can also cover GAMMA

Yep, that's the goal, hoping to discuss more at the GAMMA meeting tomorrow.

Shared this doc at today's community meeting: https://docs.google.com/document/d/1akwzBKtMKkkUV8tX-O7tPcI4BPMOLZ-gmS7Iz-7AOQE/edit. Will work on a follow up GEP once we get some feedback on the doc.

**Comment by youngnick** [comment]
/remove-lifecycle stale

@robscott, with the move of this to Memorandum status, what are your thoughts on next steps here?

**Comment by mlavacca** [comment]
/remove lifecycle-rotten

Should we keep this one open as it has been recently moved to memorandum @robscott?

**Comment by mlavacca** [comment]
/remove-lifecycle rotten

Should we keep this one open as it has been recently moved to memorandum @robscott?
